plugins/services/src/js/components/ServiceConfigEnvironmentVariablesSectionDisplay.js

```diff
@@ -59,6 +60,15 @@ class ServiceConfigEnvironmentVariablesSectionDisplay extends ServiceConfigBaseS
               }
             ];
 
+            if (editLink) {
+              columns.push({
+                heading() { return null; },
+                className: 'configuration-map-action',
+                prop: 'edit',
+                render() { return editLink; }
```

Comment:
And here is one of the reasons that I would prefer passing down the configuration rather than the component. The link element has big margins and stretches the row height of a table. Otherwise, can we perhaps try another component? Or smaller margins?

![image](https://cloud.githubusercontent.com/assets/883486/21185616/b4668bfc-c211-11e6-95b9-3b7496ab1438.png)